src/lib.rs

```diff
@@ -140,21 +139,94 @@ impl Wiki {
 
         Ok(())
     }
+    /// read all the files from current paths
+    pub fn read_files(&self, file_directory: &str) {
+        //create the default file path
+        if !Path::new(file_directory).exists(){
+            info!("Creating directory for file input: {}.", file_directory);
+            fs::create_dir(file_directory);
+        }
+        for path in self.paths.clone(){
+            let mut count = 0;
+
+            //create the path for the html site or just find it
+            let page_files_path = format!("{}/{}", file_directory, path.to_str().unwrap()).replace(".md","");
+            if !Path::new(page_files_path.as_str()).exists(){
+                info!("Creating directory for {}'s files: {}.", path.to_str().unwrap(), page_files_path.as_str());
```

Comment:
The `unwraps` can lead into panics, please avoid these by using the [display method](https://doc.rust-lang.org/std/path/struct.Path.html#method.display).